src/components/AppButton.js

```diff
@@ -0,0 +1,43 @@
+import { LitElement, html, css } from 'lit-element';
+
+class AppButton extends LitElement {
+
+  static get properties(){
+    return {
+      label: { type: String }
```

Comment:
We also have the standard to add doc blocks to each props as well.

For example:

```
       /**
       * Description for this label.
       *
       * @type {{label: String}}
       */
        label: { type: String}
```